integration/fixtures/test-images.json

```diff
@@ -4,83 +4,103 @@
         "digest": "sha256:c4b11c9b86eaba41276ae682bb6875332316242010b7523efe30f365ad0c3cb8",
         "distro": "Google Distroless",
-        "description": "Custom dpkg/status.d with text names, no apt, libssl1"
+        "description": "Custom dpkg/status.d with text names, no apt, libssl1",
+        "ignoreErrors": false
     },
     {
         "image": "quay.io/calico/cni",
         "tag": "v3.15.1",
         "digest": "sha256:a925b445c2688fc9c149b20ea04faabd40610d3304a6efda68e5dada7a41b813",
         "distro": "Redhat",
-        "description": "Valid rpm DB, microdnf & rpm present"
+        "description": "Valid rpm DB, microdnf & rpm present",
+        "ignoreErrors": false
     },
     {
         "image": "mcr.microsoft.com/cbl-mariner/base/core",
         "tag": "1.0.20220218",
         "digest": "sha256:830120b2cbfb7489c6f3270e1c74f3db0de84a4d33fecfffd427890b94d2f236",
         "distro": "Mariner",
-        "description": "Valid rpm DB, no dnf, yum & rpm present"
+        "description": "Valid rpm DB, no dnf, yum & rpm present",
+        "ignoreErrors": false
     },
     {
         "image": "mcr.microsoft.com/cbl-mariner/base/core",
         "tag": "1.0.20220218-arm64",
         "digest": "sha256:f97ccb4565f8985c28c6dbc7f13cfea0877652b70545f844eb0b83e4475954d1",
         "distro": "Mariner",
-        "description": "Valid rpm DB, no dnf, yum & rpm present, arm64 cross-arch"
+        "description": "Valid rpm DB, no dnf, yum & rpm present, arm64 cross-arch",
+        "ignoreErrors": false
     },
     {
         "image": "mcr.microsoft.com/cbl-mariner/distroless/base",
         "tag": "2.0.20220527",
         "digest": "sha256:f550c5428df17b145851ad75983aca6d613ad4b51ca7983b2a83e67d0ac91a5d",
         "distro": "Mariner Distroless",
-        "description": "Custom rpmmanifest files, no yum/dnf/microdnf/rpm"
+        "description": "Custom rpmmanifest files, no yum/dnf/microdnf/rpm",
+        "ignoreErrors": false
     },
     {
         "image": "docker.io/library/centos",
         "tag": "7.6.1810",
         "digest": "sha256:62d9e1c2daa91166139b51577fe4f4f6b4cc41a3a2c7fc36bd895e2a17a3e4e6",
         "distro": "CentOS",
-        "description": "Valid rpm DB, yum present"
+        "description": "Valid rpm DB, yum present",
+        "ignoreErrors": false
     },
     {
         "image": "docker.io/library/amazonlinux",
         "tag": "2.0.20210326.0",
         "digest": "sha256:06380711d6a8ac0b6989f7e2a4419e560796791d9c7c843753a719c73552dc30",
         "distro": "Amazon Linux",
-        "description": "Valid rpm DB, yum present"
+        "description": "Valid rpm DB, yum present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "mcr.microsoft.com/oss/nginx/nginx",
+        "tag" : "1.21.6",
+        "digest": "sha256:bcdc041d100a3cc9fec472a8133ad566166e366241587ff92e8cc8fcf955229d",
+        "distro": "Debian",
+        "description": "Valid dpkg/status, apt present",
+        "ignoreErrors": true
```

Comment:
does this fail to patch?

Reply:
yes, it fails to patch without the `ignore-errors` flag.
See the [PR description](https://github.com/project-copacetic/copacetic/pull/247#issue-1847850011) (How to Test > 2nd Step), I've also included a screenshot of it

Reply:
@anubhav06 i can't replicate this one in main branch

`docker.io/grafana/grafana-image-renderer:3.4.0` has an error with libcryto and libssl

Reply:
added

Reply:
I am surprised this is not failing in the CI